### PR TITLE
Use default LNS port for BasicStation CUPS Update Info request

### PR DESCRIPTION
**Pull request by KrishnaIyer**
#### Summary

Use default LNS port for BasicStation CUPS Update Info request.

In most of our "standard" deployments, most users choose the pre-filled GatewayServerAddress when creating a gateway. This does not have any port info so Basic Station gateways complain. 

#### Changes

- Use port 8887 in case no port is provided.

#### Notes for Reviewers

Users can just append the port in the console/cli but we should support a default port as we already support a default protocol (`wss`).
Users should only have to edit the port if they choose to customize.

#### Checklist

- [ ] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, database and configuration, according to the stability commitments in `README.md`.
- [ ] Testing: The changes are covered with unit tests. The changes are tested manually as well.
- [ ] Documentation: Relevant documentation is added or updated.
- [x] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

**Comment by adriansmares** [comment]
I think it would be better to add a new `Option` that allows the Gateway Server to propagate the listener port directly to the CUPS frontend.

**Comment by KrishnaIyer** [comment]
Yeah I thought about that but when the GS is behind a proxy, it doesn't know what the front-facing port is. We already solved that by setting the `GatewayServerAddress` to `mydomain:port` on the Gateway object.

The reason why I added this is;
* User creates a Gateway on Console/CLI using all defaults.
* User creates an API key.
* User connects gateway.
* Gateway shows error saying that the port is missing.

So this is just a sane default value that works on 90% of deployments. For custom ones, the user can always set the port manually.

File: pkg/basicstation/cups/server.go [hunk 1]
```diff
@@ -293,13 +300,10 @@ func (s *Server) getTrust(address string) (*x509.Certificate, error) {
 		}
 		return nil, errNoTrust
 	}
-	_, host, port, err := parseAddress(address)
+	_, host, port, err := parseAddress("https:", address)
```
Comment (htdvisser) [inline pkg/basicstation/cups/server.go]: I think you meant `https` without the colon

Comment (KrishnaIyer) [reply pkg/basicstation/cups/server.go]: Yup. Good catch.

**Pull request merged**